sdk/python/responsible-ai/mlflow-deployment-with-explanations/env.yml

```diff
@@ -0,0 +1,11 @@
+channels:
+  - conda-forge
+dependencies:
+  - python=3.9.13
+  - pip<=21.2.4
+  - pip:
+      - responsibleai==0.29.0
+      - ml-wrappers
+      - mlflow
```

Comment:
Can we use an existing responsibleai environment instead of defining a new environment here? The tabular environment could be found here:- https://github.com/Azure/azureml-assets/blob/main/assets/responsibleai/environments/responsibleai-ubuntu20.04-py38-cpu/context/conda_dependencies.yaml. We shouldn't be maintaining another env for deployment.

Reply:
I don't see raiutils and mlwrappers in the tabular environment - are the only text, tabular, and vision envs or are there others?

Reply:
raiutils is a dependency of the responsibleai package and therefore included everywhere. Actually, I just checked and it's just indirectly included via erroranalysis (somethign we should probably fix...) but it'll work nontheless.

This isn't obvious but we can pretty much assume that'll always stay that way. ml-wrappers is also listed: https://github.com/microsoft/responsible-ai-toolbox/blob/f6cd1f85a689e6946672721c55b5980ba8cdfa9b/responsibleai/requirements.txt#L14

Reply:
I tried use the RAI curated environment during deployment (in place of the env.yml in registration) but ran into issues. After discussing with Kin, we found that no code deployments are not compatible with custom image built with docker context.